### ISSUE TITLE
Help needed to pair account on IoT cloud

### ISSUE THREAD

**Issue by tolnai** [jasonacox/tinytuya#389]
I appreciate that this is not necessarily tinytuya related, but I tried to follow the guide here to set up my tuya IoT cloud project and I'm pulling my hair out but still can't pair my account. Any help would be appreciated.

I have a project with the proper service APIs enabled. I'm in Hungary, Hungary was selected in the Smart Life app, and Hungary is selected in my IoT account as well. This should belong in theory to the Central Europe region, but I tried with all regions in my project to scan the QR code and get an error with all of them. Previously I tried with another IoT account country, but that can only be changed every 30 days so couldn't do many tries with that.

I know it's all about the regions, but I don't get why the Central Europe region is not working, I just see the usual "invalid parameters" error in the network inspector when scanning the code... The wizard and the API explorer also didn't give me any clues. Can you help me what I'm missing here? Thanks!

**Comment by jasonacox** [comment]
It is a difficult process and any missing detail seems to break it all.  I suspect you have done this, but maybe some of this will help:
* Read the [PDF Instructions here](https://github.com/jasonacox/tinytuya/files/12101705/Tuya.IoT.API.Setup.1.pdf) and see the screenshots to make sure you setup looks good on the Tuya IoT side 
* You mention network scanning, but do try to activate the tinytuya debug option (e.g. `python3 -m tinytuya wizard -d`)
* We have seen that the "regions" that Tuya sets are really datacenters and don't always map to your real region. I think you did this already, but would try Western Europe (and others) just in case.
* Device ID is required - The wizard should scan for a deviceID on your network or you can provide it manually
* This *only* works with the `Smart Life app` but seems that is what you used.

Worse case, there are other ways to get local heys.  See this thread: https://github.com/jasonacox/tinytuya/issues/249#issuecomment-1416655241

**Comment by tolnai** [comment]
Thanks for the quick response. I went through things once more, and found the issue. 🤦
To cut it short, I used another app... It's also called Smart Life and is on play store (well, SmartLife in one word to be fair), but because the app looks almost the same, I haven't noticed. Damn those bastards for the many similar names.
Your linked comment really helped, as that was clearly telling me an invalid username/password with all data centers, so I knew something was off with my account. Registered with the proper app, re-paired my devices, and showed up with the script with the EU data center. It might help others as well if that script was more available from the main readmes.
Thanks for the help!
